src/cobalt-launcher.c

```diff
@@ -112,23 +112,36 @@ gboolean cobalt_launcher_read_flags_file(CobaltLauncher *launcher, GFile *file,
       continue;
     }
 
-    if (g_str_has_prefix(line, ENABLE_FEATURES_FLAGFILE_PREFIX) ||
-        g_str_has_prefix(line, DISABLE_FEATURES_FLAGFILE_PREFIX)) {
-      const char *feature = strchr(line, '=') + 1;
-      if (*feature == '\0') {
-        g_warning("Line in '%s' has an empty feature: %s", g_file_peek_path(file), line);
+    {
+      gint tokenc;
+      gchar **tokenv;
+      gboolean ret;
+
+      ret = g_shell_parse_argv (line, &tokenc, &tokenv, NULL);
+      if (!ret) {
+        return FALSE;
```

Comment:
It seems a bit extreme to die parsing the full file, you could probably just `g_warning`, basing it off of some of the uses below, and using a `g_autoptr(GError) local_error = NULL` and passing that would allow us to get the error message (you can also base this off of some of the other uses in the source code). (It may also be a bit cleaner to inline the value of `ret` into the `if`, instead of saving it separately.)